test/p2p/sanity.spec.ts

```diff
@@ -72,6 +80,15 @@ describe('P2P Sanity Tests', () => {
       .to.be.rejectedWith('already connected');
   });
 
+  it('should place a ownOrder', async () => {
+    await expect(nodeOne.service.placeOrder({ pairId: 'BTC/LTC', price: 600, quantity: 0.1, orderId: uuidv1() })).to.be.fulfilled;
+  });
+
+  it('should verify that the order gets transmitted between nodes' , async() => {
```

Comment:
@sangaman agree and i have noticed the behaviour. hence, i had to move placing order into its own test to make this pass. i couldn't get `timeout` function of mocha to work with typescript to delay the next test so that nodes can communicate and relay the orders / order.

Reply:
When we have streaming event of new peer orders (which should be soon, it's on my shortlist), we can wait for peerTwo to raise the event for the new order. Until then this might be good enough.